resources/design-document.md

```diff
@@ -0,0 +1,244 @@
+# Design Document
+
+## Instructions
+
+## Brecghan's Training Matrix(BTM) Design
+
+## 1. Problem Statement
+
+This service is designed to manage a training system for a company's personnel. It allows them to track active employees, what training they have received, what testing they have taken and the results of the test, and if any trainings are set to expire.
+
+This design document describes the service that will provide the training matrix functionality to meet our customers' needs. It is designed to contain employees, trainings, and a testing tracker. It will be able to return current employees, trainings that have been conducted, and tests results from the different trainings.
+
+## 2. Use Cases
+
+U1. As a BTM customer, I want to create an employee
+
+U2. As a BTM customer, I want to be able to create a training
+
+U3. As a BTM customer, I want to be able to create a training series
```

Comment:
What's the difference between a "training" and a "training series"?

Reply:
A training is a class that is attended (could be a one off of like a new system, machine, etc.) where as a Training Series is specifically for trainings that need to be continually refreshed (like a safety meeting that needs to occur every 6 months).

Reply:
Ahh... I think it'd be helpful to phrase this throughout as "recurring training series" (adding the word recurring). Unless that's a common vernacular in the training world... but seeing the word "recurring" makes it click a lot more for me.